packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -956,12 +1021,23 @@ export class ProjectService {
             explore,
         });
 
+        // TODO quick hack to get the old results on the chart with new granularity
+        // We should investigate if we can make the changes in the frontend
```

Comment:
If you want to, we can pass `oldData` to the hook on the FE, so that the BE does minimal changes on this `runViewChartQuery`. So it could only do: 
- updateMetricQueryGranularity(
            metricWithOverrideSorting,
            explore,
            granularity,
        );
        and 
-  row mapping

what do you think?

Reply:
I think we should do 2 (modify chart query in frontend) https://lightdash.slack.com/archives/C067071JKKN/p1700823674033019 and remove this fake result mapping. to ensure charts are consistent with the granularity, but we could do it on a separate PR 

We can't make changes to the metricquery in the frontend for security reasons.